src/main/scala/firrtl/Emitter.scala

```diff
@@ -325,6 +322,7 @@ class VerilogEmitter extends SeqTransform with Emitter {
      }
 
      def checkCatArgumentLegality(e: Expression): Unit = e match {
+       case _: UIntLiteral | _: SIntLiteral | _: WRef | _: WSubField =>
```

Comment:
This line appears as first statement in `checkArgumentLegality` why duplicate it here.
And just curious: is `WSubIndex` different from `WRef` and `WSubField` that it is not included here?

Reply:
This is a no-op, good catch. I will delete it. `WSubIndex` is illegal in Low FIRRTL :)

(`WSubField` is legal in Low FIRRTL for memory ports)